example_dags/transfer_comparison_with_traditional_transfer_operator.py

```diff
@@ -0,0 +1,73 @@
+import os
+from datetime import datetime
+
+from airflow import DAG
+from airflow.providers.google.cloud.transfers.s3_to_gcs import S3ToGCSOperator
+from airflow.providers.snowflake.transfers.s3_to_snowflake import S3ToSnowflakeOperator
+
+from universal_transfer_operator.constants import FileType
+from universal_transfer_operator.datasets.file.base import File
+from universal_transfer_operator.datasets.table import Table
+from universal_transfer_operator.universal_transfer_operator import UniversalTransferOperator
+
+s3_bucket = os.getenv("S3_BUCKET", "s3://astro-sdk-test")
+gcs_bucket = os.getenv("GCS_BUCKET", "gs://uto-test")
+
+
+with DAG(
```

Comment:
How much value does this example give us? We could focus on a more realistic DAG from a user perspective, such as an ELT DAG, which transfers using UTO and then transforms the loaded content, e.g. using the `SnowflakeOperator` similar to:
https://docs.astronomer.io/learn/airflow-snowflake

```
    """
    #### Transform
    Transform the forestfire_costs table to perform
    sample logic.
    """
    transform_forestfire_cost_table = SnowflakeOperator(
        task_id="transform_forestfire_cost_table",
        sql=sql_stmts.transform_forestfire_cost_table,
        params={"table_name": SNOWFLAKE_FORESTFIRE_COST_TABLE},
    )
```

Reply:
@tatiana added an example DAG: https://github.com/astronomer/apache-airflow-provider-transfers/pull/37/commits/f3caa09a2a52d1d38e25944ba044b7129b6182dd